src/read.rs

```diff
@@ -69,18 +68,11 @@ impl<R: Read> Rng for ReadRng<R> {
     fn try_fill(&mut self, dest: &mut [u8]) -> Result<(), Error> {
         if dest.len() == 0 { return Ok(()); }
         // Use `std::io::read_exact`, which retries on `ErrorKind::Interrupted`.
-        self.reader.read_exact(dest).map_err(map_err)
+        self.reader.read_exact(dest).map_err(|err|
+            Error::new(ErrorKind::Unavailable, "ReadRng: read error", Some(err)))
```

Comment:
This description doesn't really fit with the others.

Reply:
I wasn't really sure what message to give here. Note that there *should* be a chained cause (though there's no guarantee it actually gets printed).